resources/community/rocketchat.md

```diff
@@ -27,10 +19,51 @@ To learn more about the common channels, check out our [Chat Channel Conventions
+
+These types of questions are usually best suited for #devops-howto or for any of the channels relating to the specific platform tool in question. Try one of those (or both)! You might also want to try #common-components, if you're using one of the tools developed through that team!
+
+Special note: Keycloak *is* definitely a platform tool, but maintenance and support of that particular tool is being shifted to a dedicated team - make 100% sure that your keycloak issues are posted to #sso in particular, to make sure that team has a chance to see it!
+
+## I'm Having An End-User Problem
+
+Not sure how to figure out Documize privileges? Issues logging in to Rocketchat? Having trouble finding the right document on DevHub? When you're having problems with a tool you use as an end-user, make sure you include the following details:
+
+1. **What tool are you trying to use?** Make sure you let the community know which tool seems to be causing the problem.
+1. **What machine are you using when you have the problem?** Are you using a Windows machine, a Mac, an iPhone, an Android, or something else altogether? What version of the application are you using? If the problem tool is a web app, what browser are you using to access it? More detail is better - telling us that you're using Chrome is good, but telling us that you're using Chrome Version 89.0.4389.90 is even better. Have you tried to use the tool on a different machine or browser? If so, what happened?
+1. **What steps brought you to the problem?** Once again, more detail is better - telling us "I was trying to log into RC and got an error" is okay, but "I was trying to log into RC, and I got to the page where it asks me to pick IDIR or Github, but when I clicked on IDIR, the next page was an error message" is much, much better.
+1. **What exactly is the problem you're seeing?** If an error message comes up, copy-paste or screenshot the error and provide it as part of your question - they can often provide a lot of insight into the specific reason for the problem. Tip: put error messages into code blocks to make them easier to read in RC! If there isn't any kind of error message, tell us what you saw instead and why it's a bad thing. Did you find yourself on a blank white page with no content at all? Or maybe you were looking for an article on DevHub and, while the header loaded just fine, there were no words where the article is supposed to be. Or maybe you clicked a link and got taken to a page with plenty of content, but the content isn't what the link claimed it would be. What exactly went wrong? As always, more detail is better!
```

Comment:
```suggestion
1. **What exactly is the problem you're seeing?** If an error message comes up, copy-paste or screenshot the error and provide it as part of your question - they can often provide a lot of insight into the specific reason for the problem. Tip: put error messages into code blocks to make them easier to read in RC by wrapping them in three backticks (`` ``` ``)! If there isn't any kind of error message, tell us what you saw instead and why it's a bad thing. Did you find yourself on a blank white page with no content at all? Or maybe you were looking for an article on DevHub and, while the header loaded just fine, there were no words where the article is supposed to be. Or maybe you clicked a link and got taken to a page with plenty of content, but the content isn't what the link claimed it would be. What exactly went wrong? As always, more detail is better!
```